src/pdf/DayOverview/DayOverviewPdfButton.tsx

```diff
@@ -75,24 +75,24 @@ const addCongestionChargeDetailsForDayOverview = async (
     const postcodes: (string | null)[] = [];
 
     for (const parcel of parcels) {
-        if (parcel.client?.address_postcode) {
-            postcodes.push(parcel.client?.address_postcode);
-        }
+        parcel.client?.address_postcode
```

Comment:
Personal preference but I think using if/else is preferred here. Using the ternary operator is preferred when we have an expression